Feedback/forms.py

```diff
@@ -8,17 +12,15 @@
     ('5', ''),
 )
 
-		
-class FeedbackForm(forms.Form):
-	Punctuality = forms.IntegerField(label='', widget=forms.RadioSelect(
-		choices=choices), required=True)
-	Subject_knowledge = forms.IntegerField(label='', widget=forms.RadioSelect(
-		choices=choices), required=True)
-	Behavior = forms.IntegerField(label='', widget=forms.RadioSelect(
-		choices=choices), required=True)
-	Method_of_teaching = forms.IntegerField(label='', widget=forms.RadioSelect(
-		choices=choices), required=True)
-	Audibility = forms.IntegerField(label='', widget=forms.RadioSelect(
-		choices=choices), required=True)
-	Syllabus_coverage = forms.IntegerField(label='', widget=forms.RadioSelect(
-		choices=choices), required=True)
+class FeedbackForm(forms.ModelForm):
```

Comment:
use `ModelForm` directly instead of `forms.ModelForm`

Reply:
We are using `ModelFom` instead of `forms.ModelForm` because our form is directly add or change our Django model so we are avoiding duplication of our model description. Correct me if i am wrong.

Reply:
We have already imported `ModelForm` that's why we are using it. If we wouldn't import that then we could use `forms.ModelForm` also. That's it.